### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by eyalpalo**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fixed a bug where validating playbook for inputs that apear both in tasks and in inputs section returned error where 'inputs. ' was part of the playbook.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 * Added new validation to the **validate** command to fail and list all the file paths of files that are using a deprecated integration command / script / playbook.
 * **validate** will no longer fail on playbooks calling subplaybooks that have a higher `fromVersion` value, if  calling the subplaybook has `skipifunavailable=True`.
 * Fixed an issue where relative paths were not accessed correctly.
+* Fixed a bug where **validate** returned error on empty inputs not used is playbook.
```
Comment (dorschw) [inline CHANGELOG.md:12]:
```suggestion
* Fixed a bug where **validate** returned error on empty inputs not used in playbooks.
```

**Review by idovandijk — COMMENTED** [review]
Can you add in unit test?

File: demisto_sdk/commands/common/hook_validations/playbook.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def collect_all_inputs_in_use(self) -> Set[str]:
         for input in all_inputs_occurrences:
             input = input.strip()
             splitted = input.split('.')
-            if len(splitted) > 1:
+            if len(splitted) > 1 and splitted[1]:
```
[the remark this answers is not in the document]
Comment (eyalpalo) [reply demisto_sdk/commands/common/hook_validations/playbook.py:105]: >1, we want to make sure there is value in the first place in the array.

**Comment by eyalpalo** [comment]
@idovandijk do you mean unit test for the new validation? because there already is.

**Review by dantavori — APPROVED** [review]

**Pull request merged**